spec/acceptance/nodesets/windows-server-2012r2.yml

```diff
@@ -6,6 +6,7 @@ HOSTS:
     box : opentable/win-2012r2-standard-amd64-nocm
     box_url : https://vagrantcloud.com/opentable/boxes/win-2012r2-standard-amd64-nocm
     hypervisor : vagrant
+    user: vagrant
```

Comment:
Change needed: `is_cygwin: false`

Looks like this is line failing:

https://github.com/puppetlabs/beaker/blob/ba5ed3c98f5d349befbdf34c3a1a6915da76ee1d/lib/beaker/host_prebuilt_steps.rb#L283

This is where is_cygwin is defined:

https://github.com/puppetlabs/beaker/blob/f889721e5deb326a26e873bae50ff470a175134e/lib/beaker/host.rb#L180

This is where Windows::Host instance defined

https://github.com/puppetlabs/beaker/blob/f889721e5deb326a26e873bae50ff470a175134e/lib/beaker/host.rb#L46-L48

Now:

```
windows-server-amd64 17:59:28$ type C:\Windows\System32\drivers\etc\hosts
```